src/main/kotlin/no/nav/familie/ef/sak/vedlegg/VedleggService.kt

```diff
@@ -93,6 +93,7 @@ class VedleggService(
             avsenderMottaker = journalpost.avsenderMottaker,
             utsendingsinfo = journalpost.utsendingsinfo,
             tema = journalpost.tema,
+            harSaksbehandlerTilgang = dokumentInfo.dokumentvarianter?.find { it.variantformat == Dokumentvariantformat.ARKIV }?.saksbehandlerHarTilgang ?: false
```

Comment:
Default er arkiv. Gjorde det slik at dersom det ikke finnes noe dokument av typen arkiv så får ikke saksbehandler tilgang. For jeg så et annet sted i koden at dokumentet antakeligvis ikke er ferdig dersom status ikke har blitt arkiv, og får beskjed om at det må åpnes i gosys i stedet